### PR TITLE
Tutorial update

### PR DESCRIPTION
**Pull request by mitjmcc**
- New button canvas prefabs
- Buttons placed in new places
- New button hot swap script
- New keyboard+mouse images

### CODE REVIEW

File: Assets/Scripts/UI/InputImageHotSwap.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		
+		void Update ()
+		{
+			if (CustomInput.UsingPad != usingPad)
+			{
+				print("input change");
+				usingPad = CustomInput.UsingPad;
+				if (usingPad)
+				{
+					print("controller change");
```
Comment (JonathanHunter) [inline Assets/Scripts/UI/InputImageHotSwap.cs]: Don't include debug statements

File: Assets/Scripts/UI/InputImageHotSwap.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+				print("controller set on start");
+				SetImage(1);
+			}
+		}
+		
+		void Update ()
+		{
+			if (CustomInput.UsingPad != usingPad)
+			{
+				print("input change");
```
Comment (JonathanHunter) [inline Assets/Scripts/UI/InputImageHotSwap.cs]: Don't include debug statements

File: Assets/Scripts/UI/InputImageHotSwap.cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		private SpriteRenderer image;
+		private bool usingPad;
+
+		void Start ()
+		{
+			image = GetComponentInChildren<SpriteRenderer>();
+			usingPad = CustomInput.UsingPad;
+			if (CustomInput.UsingPad) 
+			{
+				print("controller set on start");
```
Comment (JonathanHunter) [inline Assets/Scripts/UI/InputImageHotSwap.cs]: Don't include debug statements

File: Assets/Scripts/UI/InputImageHotSwap.cs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+						SetImage(1);	// General parts of a controller
+					else if (CustomInput.GamePadType(0) == CustomInput.ControlType.Xbox)
+						SetImage(1);	// Xbox buttons
+					else if (CustomInput.GamePadType(0) == CustomInput.ControlType.PS3
+						  || CustomInput.GamePadType(0) == CustomInput.ControlType.PS4)
+						SetImage(2);	// PS buttons
+				} 
+				else
+				{
+					print("keyboard change");
```
Comment (JonathanHunter) [inline Assets/Scripts/UI/InputImageHotSwap.cs]: Don't include debug statements

**Review by JonathanHunter — CHANGES_REQUESTED** [review]

**Comment by mitjmcc** [comment]
Fixed [here](https://github.com/JonathanHunter/GGJ2018/pull/6/commits/f0241691fd14a617430d286839705c43fbb2ef60)

File: Assets/Scripts/UI/InputImageHotSwap.cs (hunk 4 above)
Comment (mitjmcc) [reply Assets/Scripts/UI/InputImageHotSwap.cs]: Fixed [here](https://github.com/JonathanHunter/GGJ2018/pull/6/commits/f0241691fd14a617430d286839705c43fbb2ef60)

File: Assets/Scripts/UI/InputImageHotSwap.cs (hunk 1 above)
Comment (mitjmcc) [reply Assets/Scripts/UI/InputImageHotSwap.cs]: Fixed [here](https://github.com/JonathanHunter/GGJ2018/pull/6/commits/f0241691fd14a617430d286839705c43fbb2ef60)

File: Assets/Scripts/UI/InputImageHotSwap.cs (hunk 2 above)
Comment (mitjmcc) [reply Assets/Scripts/UI/InputImageHotSwap.cs]: Fixed [here](https://github.com/JonathanHunter/GGJ2018/pull/6/commits/f0241691fd14a617430d286839705c43fbb2ef60)

File: Assets/Scripts/UI/InputImageHotSwap.cs (hunk 3 above)
Comment (mitjmcc) [reply Assets/Scripts/UI/InputImageHotSwap.cs]: Fixed [here](https://github.com/JonathanHunter/GGJ2018/pull/6/commits/f0241691fd14a617430d286839705c43fbb2ef60)

**Review by JonathanHunter — APPROVED** [review]

**Pull request merged**